src/java.base/share/classes/jdk/internal/classfile/impl/verifier/VerificationWrapper.java

```diff
@@ -129,6 +131,10 @@ String descriptor() {
             return m.methodType().stringValue();
         }
 
+        String parameters() {
```

Comment:
We should just use the mtd's displayDescriptor, a class file can have bridge methods with covariant return types and the bridge may be broken while the concrete method is ok.

Reply:
I preferred readability of the error messages over full information. 
Broken covariant is a corner case and exact method can be found even from the reduced info provided.